client/src/components/Forms/signup/index.jsx

```diff
@@ -0,0 +1,181 @@
+/* eslint-disable no-console */
+import React, { useState } from 'react';
+import axios from 'axios';
+import {
+  Modal,
+  Button,
+  Form,
+  Input,
+  message,
+} from 'antd';
+import logo from '../../../assets/images/logo.png';
+import './index.css';
+
+const { Item } = Form;
+const { Password } = Input;
+
+function Signup() {
+  // eslint-disable-next-line no-unused-vars
+  const [isModalVisible, setIsModalVisible] = useState(false);
+  const [fullName, setFullName] = useState('');
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [confirm, setConfirm] = useState('');
+
+  const signUp = async () => {
+    try {
+      await axios.post('/api/signup', {
+        fullName,
+        email,
+        password,
+      });
+      setIsModalVisible(false);
+      message.success('Welcome to Nova');
```

Comment:
There is a message sent from the back to tell the users that they need to verify their accounts so receive this message and show it.